cli.js

```diff
@@ -59,7 +59,7 @@ if (appName) {
 if (input) {
 	await open(input, options);
 } else {
-	const stdin = await getStdin.buffer();
+	const stdin = await streamConsumers.buffer();
```

Comment:
This isn't the way `streamConsumers.buffer()` is meant to be used. It requires a stream as an argument ([docs](https://nodejs.org/api/webstreams.html#streamconsumersbufferstream)).

Reply:
crap I somehow missed this despite already contributing a similar fix elsewhere

my apologies, will fix the commit shortly